spec/unit/ExasolWebsocket_spec.lua

```diff
@@ -69,8 +69,15 @@ describe("ExasolWebsocket", function()
         end)
 
         it("returns error when socket returns error", function()
+            -- We are suppressing STDERR here to avoid having something in the
+            -- test log that looks like a failure, but isn't.
+            -- Note that we also need to suppress LuaCheck error 122 in this case.
+            local original_stderr = io.stderr
+            finally(function() io.stderr = original_stderr end) -- luacheck: ignore 122
+
             simulate_socket_error("mock error")
             local result, err = exa_socket:send_login_command()
+            io.stderr = original_stderr -- luacheck: ignore 122
             assert.is_nil(result)
             assert.is_same("mock error", tostring(err))
```

Comment:
It does not look like you replace stderr at all? Do you mean to run `io.stderr = nil` or similar?

Reply:
You are right. I fixed that.